src/test/java/seedu/address/model/event/DateTimeTest.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.model.event;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class DateTimeTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new DateTime(null));
+    }
+
+    @Test
+    public void constructor_invalidDate_throwsIllegalArgumentException() {
+        String invalidDate = "";
+        assertThrows(IllegalArgumentException.class, () -> new DateTime(invalidDate));
+    }
+
+    @Test
+    public void isValidEmail() {
```

Comment:
```suggestion
    public void isValidDateTime() {
```

Comments about the invalid datetime mentioned above could be applied here as well.